experiments/LSM/ozark/ozark.jl

```diff
@@ -175,8 +176,19 @@ end
 update_aux! = make_update_aux(land)
 update_aux!(p, Y, 0.0)
 
-# Set up timestepper
-ode_algo = CTS.ExplicitAlgorithm(timestepper)
+# Set up timestepper and jacobian for soil
+update_jacobian! = make_update_jacobian(land.soil)
+
+ode_algo = CTS.IMEXAlgorithm(
+    timestepper,
+    CTS.NewtonsMethod(
```

Comment:
For this test, could we use max iters = 10 and the convergence criterion? That may be the more likely use scenario.